packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -180,3 +180,149 @@ bool _assertOutOfExtent(double extent) {
   }
   return true;
 }
+
+/// A sliver that places multiple sliver children in a linear array along the
+/// main axis.
+///
+/// The layout algorithm lays out slivers one by one. If the sliver is visible
+/// on the [Viewport] due to a sufficiently high [SliverConstraints.scrollOffset],
+/// then we compute a valid [SliverConstraints.scrollOffset] and
+/// [SliverConstraints.remainingPaintExtent] based on the total [SliverConstraints.scrollOffset]
+/// of all the sliver children laid out so far.
+///
+/// Finally, this sliver will also ensure that all slivers are painted within
+/// the total scroll extent of the sliver by adjusting the children's
+/// [SliverPhysicalParentData.paintOffset] as necessary. This can happen for
```

Comment:
```suggestion
/// Finally, this sliver will also ensure that all child slivers are painted within
/// the total scroll extent of the group by adjusting the child's
/// [SliverPhysicalParentData.paintOffset] as necessary. This can happen for
```